examples/meson.build

```diff
@@ -90,6 +90,14 @@ examples = {
 		'src': 'toplevel-decoration.c',
 		'dep': [wayland_client, wlr_protos, wlroots],
 	},
+	'input-method': {
+	    'src': 'input-method.c',
+	    'dep': [wayland_client, wlr_protos, wlroots],
+    },
+	'text-input': {
+	    'src': 'text-input.c',
+	    'dep': [wayland_cursor, wayland_client, wlr_protos, wlroots],
+    },
```

Comment:
You're adding both in the same commi alhough one should be in text-input and the other one in input-method.

Reply:
Should be fixed now.